README.md

```diff
@@ -102,3 +102,17 @@ Then, from within your project folder:
 npm run build
 surge public my-project.surge.sh
 ```
+
+### With [Netlify](https://www.netlify.com/)
+
+Go to https://www.netlify.com/ and create a free account.
```

Comment:
The Vercel and surge instructions assume you have an account. We should probably do the same here for consistency and conciseness